packages/grafana-ui/src/components/IconButton/IconButton.story.tsx

```diff
@@ -19,9 +19,23 @@ const meta: ComponentMeta<typeof IconButton> = {
       page: mdx,
     },
   },
+  args: {
+    name: 'apps',
+    size: 'md',
+    iconType: 'default',
+    tooltip: 'sample tooltip message',
+    tooltipPlacement: 'top',
+    variant: 'secondary',
+    ariaLabel: 'sample aria-label content',
+  },
+  argTypes: {
+    tooltip: {
+      control: 'text'
```

Comment:
It is missing a comma at the end of this line which is throwing an error on prettier, will just add that and then we can merge 😄 
```suggestion
      control: 'text',
```